Jenkinsfile

```diff
@@ -1,42 +1,73 @@
 #!groovy
 
-def supportedPhpVersions = ["5.6", "7.0", "7.1"]
+String launchUnitTests = "yes"
+String launchIntegrationTests = "yes"
+String pimVersion = "1.7"
 
-def launchUnitTests = "yes"
-def launchIntegrationTests = "no"
-def clients = ["php-http/guzzle5-adapter", "php-http/guzzle6-adapter"]
-// def pimVersion = "1.7"
+def clientConfig = [
+    "php-http/guzzle6-adapter": ["phpVersion": ["5.6", "7.0", "7.1"], "psrImplem": ["guzzlehttp/psr7"]],
```

Comment:
Why not using `supportedPhpVersions` here (and below)? You would have only one line to change when you'll had PHP 7.2.